manage

```diff
@@ -708,22 +719,30 @@ Or on Discord: \e[94m\e[4mhttps://discord.gg/RXSTvaUvuu\e[0m" | tee -a "$logfile
   #exit the manage script with the same exit-code of the app's script
   exit $exitcode
   
-elif [ "$1" == 'update' ] && [[ "$2" =~ ^filelist:.* ]];then #list of files separated by :
-  # use the second argument to pass our list of files to update
-  #get functions from updater script
-  source "${DIRECTORY}/updater" source
-
-  updatable_apps='' updatable_files="$(echo "$2" | sed 's/^filelist://g' | tr ':' '\n')" no_status=true update_now_cli
+elif [ "$1" == 'refresh' ];then #copy the online version of an app to the main pi-apps folder. Used when an app does not need to be reinstalled when updating it/
+  status -n "Refreshing \e[1m${app}\e[0m\e[96m... "
+  #Set terminal title
+  echo -ne "\e]0;Refreshing ${app}\a"
+  
+  #move old program to trash - this allows app-developers to recover their work if update was accidental
+  gio trash "${DIRECTORY}/apps/${app}" 2>/dev/null
+  #failsafe
+  rm -rf "${DIRECTORY}/apps/${app}"
+  
+  #copy new version from update/ to apps/
+  cp -rf "${DIRECTORY}/update/pi-apps/apps/${app}" "${DIRECTORY}/apps/${app}"
   
-elif [ "$1" == 'update' ] || [ "$1" == 'refresh' ];then
+  status_green Done
```

Comment:
this isn't very descriptive. is there a reason `status_green` was changed from
```
  if [ "$failed" == 'true' ]; then
    echo -e "\e[91mFailed to update ${app}.\e[0m"
    return 1
  else
    status_green "${app} was updated successfully."
    return 0
  fi
  ```
 ?

Reply:
Refreshing many apps at once is common. Since refreshing an app never generates any output, there is no need for large title blocks at the beginning and end. Rather, each refreshed app generates just one line in terminal: the status and the completion message at the end. (which I should improve to say "failed" if it fails)